external/multi-base-images/spark/releaseCloudflowSpark.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+SCRIPT=`basename ${BASH_SOURCE[0]}`
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd -P )"
+TAG=v2.4.4
+ORIGIN_TAG=2.4.4-k8s-client-upgrade
+DOCKER_USERNAME=lightbend
+CLOUDFLOW_VERSION=1.3.1-SNAPSHOT
+SPARK_IMAGE_TAG=${CLOUDFLOW_VERSION}-cloudflow-spark-2.4.4-scala-2.12
+SPARK_OPERATOR_TAG=${CLOUDFLOW_VERSION}-cloudflow-spark-2.4.4-1.0.1-scala-2.12
```

Comment:
@yuchaoran2011 @debasishg I am changing image names to something more useful and more in alignment with #121 .